### PR TITLE
Fix 404 issue for some idols

### PR DESCRIPTION
**Pull request by KaiDevrim**
Refer to #14 for more info also

Basically just use fields.slug for guests and also add nvmrc. Also, the port is specified because despite the port 8080 being added to lambda, it did not work for until I specified port 8000. This also deletes the bad-slugged/named guest pages and replaces them with the correctly named versions. (Generated from the cms).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
     "start": "npm run develop",
     "start-compiled": "npm run build && npm run start-static-server",
-    "start-static-server": "npx http-server public",
+    "start-static-server": "npx http-server public -p 8000",
```
Comment (saralain) [inline package.json]: Do we actually need to do this? I find it valuable to have the static server on a different port, so I can run the two side-by-side at times. Plus, it makes it easy to spot which one you're looking at. 

Open for discussion if there's a good reason

File: static/admin/config.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ collections:
     folder: "src/markdown-pages/about"
     create: true
     delete: true
-    slug: "{{slug}}" # (    o_o) slug
+    slug: "{{fields.slug}}" # (    o_o) slug
```
Comment (saralain) [inline static/admin/config.yml]: This appears to only affect pages in the about section - however the problem we're experiencing is with guests. Can we move this to the guests section of the config file?

**Review by saralain — COMMENTED** [review]
I think we're really close! I did point out one thing I think needs attention. 

I'll also ask that we squash these commits when we merge, just so we don't have a bunch of noop guest updates in history. (Not the end of the world if we forget sometimes - git history is already a huge mess with my prior work)

File: static/admin/config.yml (hunk 2 above)
Comment (KaiDevrim) [reply static/admin/config.yml]: oh yeah i put in the wrong section, will fix

File: package.json (hunk 1 above)
Comment (KaiDevrim) [reply package.json]: makes sense will turn back to normal

**Comment by KaiDevrim** [comment]
Yeah feel free to squash it, I also just addressed everything. Now it should be good to go.

**Comment by saralain** [comment]
Thanks for taking care of this!

**Pull request merged**